platform/ATCmdParser.h

```diff
@@ -20,9 +20,10 @@
 #ifndef MBED_ATCMDPARSER_H
 #define MBED_ATCMDPARSER_H
 
-#include "mbed.h"
 #include <cstdarg>
 #include "Callback.h"
+#include "NonCopyable.h"
+#include "FileHandle.h"
```

Comment:
The latest build points to this file, seems like some includes are missing and build fails

Reply:
Fixed: std namespace was missing